packages/web/lib/logging/server-simple.ts

```
@@ -0,0 +1,70 @@
+      case 'warn':
+        console.warn(logEntry)
+        break
+      default:
+        console.log(logEntry)
+    }
+  }
+
+  return {
+    error: (message: string, metadata?: LogMetadata) => {
+      if (metadata?.error) {
+        metadata = {
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)